Mathlib/Algebra/Module/LocalizedModule.lean

```diff
@@ -1110,4 +1110,61 @@ theorem isBaseChange : IsBaseChange A f := by
 
 end IsLocalizedModule
 
+section Basis
+
+open Submodule
+
+variable (Rₛ : Type*) [CommRing Rₛ] [Algebra R Rₛ] [IsLocalization S Rₛ] (Mₛ : Type*)
+  [AddCommGroup Mₛ] [Module R Mₛ] {M : Submodule R Mₛ} [IsLocalizedModule S M.subtype]
+  [Module Rₛ Mₛ] [IsScalarTower R Rₛ Mₛ]
+
+theorem SpanEqTop.isLocalizedModule {v : Set M} (hv : span R v = ⊤) :
+    span Rₛ (M.subtype '' v) = ⊤ := by
```

Comment:
I just copied what was done there: [SpanEqTop.localization_localization](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/Module.html#SpanEqTop.localization_localization) but I am not a big fan of the name, I agree.

Reply:
@Vierkantor you added that [in mathlib3](https://github.com/leanprover-community/mathlib/commit/b69c9a770ecf37eb21f7b8cf4fa00de3b62694ec#diff-73f8ec4033c2c85254b321f25ca5d5360339bee6450f3ff4ad8b68dab89b2b7bR77-R78). Do you agree we should rename it to `_of_span_eq_top`?

Reply:
IIRC, the motivation was for symmetry with `LinearIndependent`. I'm happy with either version, so please go ahead and rename!

Reply:
Thanks! LinearIndependent makes dot notation available, but SpanEqTop dot notation doesn't work unless we actually make a def SpanEqTop. Since it's unclear whether we want to do that, and given that [SpanEqTop.localization_localization](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/Module.html#SpanEqTop.localization_localization) is currently the only declaration in the namespace, I think we should rename it.

Reply:
But should not the names be more like `span_eq_top_of_` than the other way around?